evm_arithmetization/src/arithmetic/mod.rs

```diff
@@ -26,7 +26,7 @@ pub(crate) mod columns;
 /// `Shl` and `Shr` are handled differently, by leveraging `Mul` and `Div`
 /// respectively.
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
-pub(crate) enum BinaryOperator {
+pub enum BinaryOperator {
```

Comment:
why this change?

Reply:
A new integration test, add11_yml_first_segment, is added in this PR, which verifies that no Keccak operations is in the segment data.

Reply:
I’ll do some refactoring to revert this change.

Reply:
Done

Reply:
QQ: why adding an integration test if we already have a unit test for this? (and why in `add11_yml` which is a specific existing test coming from Ethereum test suite)

Reply:
I’ve already moved it out of `add11_yml`. They serve different purposes for testing the two distinct ‘prove all segment’ functions, as we discussed offline. I’ll work on unifying them into one test.